src/fundus/publishers/de/netzpolitik_org.py

```diff
@@ -0,0 +1,50 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_date_parsing,
+    generic_topic_parsing,
+    parse_title_from_root,
+    strip_nodes_to_text,
+)
+
+
+class NetzpolitikOrgParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div.entry-content > p")
```

Comment:
Judging by [this article](https://netzpolitik.org/2024/mastodon-und-peertube-eu-macht-fediverse-dicht-weil-niemand-die-server-betreiben-will/) the selector seems to have problems extracting text wrapped in `blockquote` tags.